core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/cookie/CookieComparator.java

```diff
@@ -118,43 +125,56 @@ private ComparatorStepResult testCookie(Set<Cookie> cookies) {
     CookieTestComparatorResult result = new CookieTestComparatorResult(compareAction, cookies, name,
         value);
     final String artifactId = artifactsDAO.saveArtifactInJsonFormat(properties, result);
-    return new ComparatorStepResult(artifactId,
-        testResult ? ComparatorStepResult.Status.PASSED : ComparatorStepResult.Status.FAILED);
+    ComparatorStepResult stepResult = new ComparatorStepResult(artifactId,
+        testResult ? Status.PASSED : Status.FAILED);
+
+    stepResult.addData("compareAction", compareAction.name());
+
+    return stepResult;
   }
 
   private ComparatorStepResult listCookies(Set<Cookie> cookies) {
     CookieComparatorResult result = new CookieComparatorResult(compareAction, cookies);
     final String artifactId = artifactsDAO.saveArtifactInJsonFormat(properties, result);
-    return new ComparatorStepResult(artifactId, ComparatorStepResult.Status.PASSED);
+    ComparatorStepResult stepResult = new ComparatorStepResult(artifactId, Status.PASSED);
+    stepResult.addData("compareAction", compareAction.name());
```

Comment:
Could you please move `"compareAction"` to constant and reuse in all places?

Reply:
Fixed.